studio/components/layouts/AuthLayout/AuthLayout.utils.ts

```diff
@@ -15,7 +15,7 @@ export const generateAuthMenu = (ref: string): ProductMenuGroup[] => {
           ? [
               {
                 name: 'Logs',
-                key: 'logs',
+                key: 'auth-logs',
```

Comment:
I noticed the logs menu item wasn't highlighted when active because this key didn't match the URL, so I did a quick fix here. Is this key used anywhere else that also needs updating to match?